### PR TITLE
MAINTAINERS: update ASoC/Intel/SOF maintainers

### PR DESCRIPTION
**Pull request by plbossart**
Keyon Jie was a key contributor to the Intel ASoC and SOF Intel
drivers, but he's moved on to a different role within Intel. We wish
him all the best in his new endeavors.

Bard Liao, Kai Vehmanen, Ranjani Sridharan and Peter Ujfalusi have
been involved in the Intel multi-maintainer team, it's time to update
the MAINTAINERS entry to reflect their contributions and clarify their
role.

Signed-off-by: Pierre-Louis Bossart <pierre-louis.bossart@linux.intel.com>

### CODE REVIEW

**Review by bardliao — APPROVED** [review]

**Comment by plbossart** [comment]
@ranj063 @ujfalusi I can't merge this if you don't agree :-)

**Comment by plbossart** [comment]
@ujfalusi ping

**Review by ujfalusi — APPROVED** [review]

**Pull request merged**
